src/opendr/perception/pose_estimation/hr_pose_estimation/HighResolutionLearner.py

```diff
@@ -0,0 +1,992 @@
+# Copyright 2020-2022 OpenDR European Project
```

Comment:
File needs to be renamed to snake_case: `HighResolutionLearner.py` -> `high_resolution_learner.py`